src/components/ArticleHeadings/hooks.js

```diff
@@ -35,8 +35,14 @@ export function useSidenavItems(tableOfContents) {
       tableOfContents,
       'text/html'
     )
-    parseItems(parsedHtml.getElementsByTagName('ul'))
-
+    const pCount = parsedHtml.getElementsByTagName('p').length
+    const ulCount = parsedHtml.getElementsByTagName('ul').length
+    if (pCount === 1 && ulCount === 2) {
+      parseItems([parsedHtml.getElementsByTagName('ul')[1]])
+    } else {
+      parseItems(parsedHtml.getElementsByTagName('ul'))
+    }
+    
```

Comment:
Store retrieved elements in the variable, e.g. `const ul = parsedHtml.getElementsByTagName('ul')`